docs/framework/winforms/windows-forms-accessibility-improvements.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Windows Forms Accessibility Improvements"
+ms.date: "08/16/2019"
+helpviewer_keywords: 
+  - "Windows Forms accessibility"
+  - "accessibility"
+author: "M-Lipin"
+---
+# Accessibility improvements in Windows Forms controls for .NET Core 3.0
+
+Windows Forms is continuing to improve how it works with accessibility technologies to better support Windows Forms customers. These include the following changes:
+- Changes in various areas of interaction with accessibility client applications, including Narrator.
+- Changes in the Accessible hierarchy (improving navigation through the UI Automation tree).
+- Changes in keyboard navigation.
+
+- [x] The quirks which were used in .NET Framework 4.7.1 - 4.8 to opt-in/out new accessibility features have been removed, so previously quirked behavior is now enabled by default.
+
+### Recommended Action
+__The changes which could be opted in or out in .NET Framework 4.7.1 - 4.8 are now enabled by default and cannot be opted out__
+  
+Starting with .NET Core 3.0, Windows Forms applications benefit from all the new accessibility features (introduced in .NET Framework 4.7.1 - 4.8) without additional configuration.
+
+## Improved ListBox Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the xref:System.Windows.Forms.ListBox control:
+- Enabled UIA support for ListBox control.
+- Improved ListBox accessibility support by adding the ScrollItem pattern to ListBox items and by enhancing the accessibility event raising and handling and Narrator navigation through the items (caps lock navigation is not correct and does not throw the navigation outside the control unintentionally).
+
+## Improved CheckedListBox Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the xref:System.Windows.Forms.CheckedListBox control:
+- Corrected CheckedListBox Bounds provided by accessibility properties for entries. Improved overall ListBox and CheckedListBox accessibility (corrected pproperty values and event model).
+
+## Improved ComboBox Accessibility support
+On .NET Core, Windows Forms includes the following changes that affect the xref:System.Windows.Forms.ComboBox control:
```

Comment:
These refs need to be fixed, they are actually XML entries:

```suggestion
On .NET Core, Windows Forms includes the following changes that affect the <xref:System.Windows.Forms.ComboBox> control:
```

You can use this format for all controls referenced in this article.

Reply:
Fixed this, thanks!